### ISSUE TITLE
what is the difference sft data and rl data?

### ISSUE THREAD

**Issue by jiangyigithub** [ucla-mobility/AutoVLA#35]
It seems that there is only one script for generating the training dataset:

bash scripts/run_nuplan_preprocessing.sh

I have a few questions:

1.Are both the SFT data and RL data generated by this unified preprocessing script?
2.What are the key differences in how SFT data and RL data are constructed?
3.How do we determine the appropriate SFT-to-RL data ratio when training on the full dataset, such as WOD-E2E and nuPlan?

**Comment by Zewei-Zhou** [comment]
Hi, here are our answers to your questions:

1. Yes, all training data from different sources are unified into a standardized JSON format and can be loaded using a single, unified dataloader for both SFT and RL. The primary difference is that RL data requires integration with an external evaluation pipeline (e.g., PDMS of NAVSIM). You need to run a metric cache of NAVSIM beforehand so that the rewards can be calculated properly during training.

2. Determining the optimal SFT-to-RL data ratio is still an open question in the community, and exploring how to optimize this balance is valuable. Currently, we haven't deeply investigated this aspect and simply use a straightforward random selection method.
